app/components/CredentialCard.tsx

```diff
@@ -69,20 +70,17 @@ export default function CredentialCard({
               />
               Delete
             </ModalToggleButton>
-            <Form
-              method="GET"
-              action="/quickstart/alerts"
-              className="display-inline"
+            <Link
+              to={`/quickstart/alerts?clientId=${client_id}&${searchParams.toString()}`}
```

Comment:
It's fine to change this from a Button to a Link, but don't have a Button _inside of a link_. You can make a Link appear as a Button by giving it the `usa-button` style. (You probably need to make sure that it is a React Link and not a React-USWDS Link which would have unwanted baked-in classes).